config.php.sample

```diff
@@ -184,6 +184,7 @@ $u5samllogoutresultpage = 'https://test.example.com/index.php?c=farewell';  // w
 $u5samlautointranetenrollment = 'no';
 $u5samlmockauth = 'no';
 $u5samlmockauthusername = 'xavier.yalonden@example.com';
+$u5samlinfrontendyesenforcedifloginformsgetudoesnotcontain='thatstring';
```

Comment:
If adding this config variable, shouldn't you then also add the corresponding $u5samlinfrontend that is currently not in the sample config but only in your documentation as it is something not widely used?

Reply:
how true...